components/Main/Exercises/Favorite/Favorite.js

```diff
@@ -0,0 +1,39 @@
+import Image from "next/image";
+import { useState } from "react";
+import { StyledFav } from "./StyledFav";
+
+export default function Favorite() {
+  const [favourite, setFavourite] = useState(false);
```

Comment:
I assume you will later have to uplift this state to a higher level, to be able to conditional render the favorites, depending on this state? But right now, you "only" toggle the buttons with it, which ist fine.

Reply:
Thäänks

Reply:
Yes will do this in the next US.